hyperspy/utils/__init__.py

```diff
@@ -85,3 +85,21 @@ def print_known_signal_types():
         table.sortby = "class name"
     return print_html(f_text=table.get_string,
                       f_html=table.get_html_string)
+
+
+__all__ = [
+    'eds',
+    'interactive',
+    'material',
+    'model',
+    'plot',
+    'print_known_signal_types',
+    'roi',
+    'samfire',
+    'stack',
+    'transpose',
+    ]
+
+
+def __dir__():
```

Comment:
There are a number of such codecov warnings

Reply:
Yes, I need to add tests as done in https://github.com/hyperspy/hyperspy_gui_ipywidgets/pull/40 and https://github.com/hyperspy/hyperspy_gui_traitsui/pull/41